tests/molecule_benchmark_test.go

```diff
@@ -2,19 +2,18 @@ package moleculetest
 
 import (
 	"testing"
-	"time"
 
 	"github.com/richardartoul/molecule"
 	"github.com/richardartoul/molecule/src/codec"
-	"github.com/richardartoul/molecule/src/proto"
+	simple "github.com/richardartoul/molecule/src/proto"
 
 	"github.com/golang/protobuf/proto"
-	"github.com/google/gofuzz"
+	fuzz "github.com/google/gofuzz"
 )
 
 func BenchmarkMolecule(b *testing.B) {
 	var (
-		seed   = time.Now().UnixNano()
+		seed   = int64(1623963202)
```

Comment:
Can you set this back to a random seed? I like getting fresh data on each C.I run for free fuzz testing

Reply:
Nevermind I see this is a benchmark, this is fine then